test_cases/integer_overflow_and_underflow/integer_overflow_multitx_onefunc_infeasible.yaml

```diff
@@ -0,0 +1,4 @@
+description: "Integer overflow (unreachable)"
+issues:
+- id: "SWC-101"
```

Comment:
For test samples that have benign cases only, we still specify `id` and `count` so that the  configuration is explicit in terms of what weaknesses the analysis tools is supposed to look for.   So in this case it would only look for instances of SWC-101 and the expectation is that it finds 0. Test case runners will measure performance in that way. If the tool finds other issues in the test sample then they will be disregarded. This is advantages when real life contracts are added and only specific weaknesses are of interest.

Reply:
Ok, that makes sense.

Reply:
Should it also have "count: 0" then?

Reply:
It does have `count: 0`